src/olympia/addons/views.py

```diff
@@ -538,11 +538,20 @@ def icloud_bookmarks_redirect(request):
         return addon_detail(request, 'icloud-bookmarks')
 
 
+DEFAULT_FIND_REPLACEMENT_PATH = '/collections/mozilla/featured-add-ons/'
+FIND_REPLACEMENT_SRC = 'find-replacement'
```

Comment:
This `src` needs to be added to the list of valid sources via a migration - see `DownloadSource` model in `zadmin`. Also needs to be added to the API docs (I hardcoded the valid sources on prod there so that the frontend can use them as well)